src/common/Finisher.cc

```diff
@@ -12,7 +12,7 @@
 void Finisher::start()
 {
   ldout(cct, 10) << __func__ << dendl;
-  finisher_thread.create();
+  finisher_thread.create("finisher");
```

Comment:
Yeah.. I'd say add a second arg to the workqueue so that you have to specify both a long name and a short name.